.github/workflows/release.yml

```diff
@@ -18,6 +18,36 @@ jobs:
       - name: Run tests
         run: make unit-tests
 
+  docs:
+    needs: ["test"]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: install Doxygen 1.9.3
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y graphviz libclang-cpp1-9 libclang1-9
+          wget https://www.doxygen.nl/files/doxygen-1.9.3.linux.bin.tar.gz
```

Comment:
I used [mattnotmitt/doxygen-action@v1 action](https://github.com/mattnotmitt/doxygen-action) but looking at this code it seems we could do the same on mobile as it's very simple.